cmd/sign-off-checker/main.go

```diff
@@ -17,27 +17,23 @@ limitations under the License.
 package main
 
 import (
-	"context"
-	"fmt"
 	"log"
 	"net/http"
 	"os"
-	"regexp"
+	"strings"
+	"time"
 
 	"github.com/google/go-github/github"
 	"golang.org/x/oauth2"
-)
-
-var client *github.Client
-var secret []byte
 
-var testRE *regexp.Regexp
+	"github.com/heptio/sign-off-checker/pkg/register"
+	"github.com/heptio/sign-off-checker/pkg/webhook"
+)
 
-func init() {
-	testRE = regexp.MustCompile(`(?mi)^signed-off-by:`)
-}
+// How often do we loop through and autoregister webhooks and branch protection configuration?
+var autoregisterInterval = 10 * time.Minute
 
-func loggingMiddleware(handler http.Handler) http.Handler {
+func loggingMiddleware(log *log.Logger, handler http.Handler) http.Handler {
```

Comment:
NB: At some point ~soon we'll want to standardize on the error reporting and logging stuff that @ncdc has been working on.  Probably doable after this though.

Reply:
+1, I'll do a followup PR to swap this over to logrus and add in graceful shutdown and liveness checks using the other libraries I've been working on.